charts/README.md

```diff
@@ -0,0 +1,88 @@
+# Sensitive Data Archive Helm Charts
+
+<!-- Developer Comment: please keep in mind that contents below will appear as inline markdown sections in the NeIC SDA Handbook guide [Deploying on Kubernetes](https://neic-sda.readthedocs.io/en/latest/guides/deploy-k8s/) page. -->
+
+## Charts overview
+
+The `neicnordic` Helm repository contains the following charts (for configuration details click on the links below):
+
+- [sda-svc - SDA services](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-svc/README.md)
+
+  This chart deploys the service components needed to operate the Sensitive Data Archive solution. The charts may include additional service components that might be beneficial for administrative operations or extending the Sensitive Data Archive solutions to facilitate different use cases.
+
+- [sda-db - SDA database](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-db/README.md)
+
+  This chart deploys a pre-configured database ([PostgreSQL](https://www.postgresql.org/)) instance for Sensitive Data Archive, the database schemas are designed to adhere to [European Genome-Phenome Archive](https://ega-archive.org/) federated archiving model.
+
+- [sda-mq - SDA Message broker](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-mq/README.md)
+
+  This chart deploys a pre-configured message broker ([RabbitMQ](https://www.rabbitmq.com/)) designed for [European Genome-Phenome Archive](https://ega-archive.org/) federated messaging between `CentralEGA` and Local/Federated EGAs but also configurable to support Standalone SDA deployments.
+
+- [sda-orch - SDA orchestrate service](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-orch/README.md)
+
+  This chart deploys an orchestration service for the Sensitive Data Archive solution. This is a helper service designed to curate the ingestion flow in an automated manner when the SDA solution is deployed and configured as standalone (non-federated).
```

Comment:
This chart has not been tested for ages, so it might be non functional by now.

Reply:
Yep, I was wondering about that but thought to add it since the chart is there. I added a warning about this as a note.